api/courses.go

```diff
@@ -34,6 +36,15 @@ func configGinCourseRouter(router *gin.Engine, daoWrapper dao.DaoWrapper) {
 
 	api := router.Group("/api")
 	{
+		api.GET("/courses/live", routes.getLive)
+		api.GET("/courses/public", routes.getPublic)
+		api.GET("/courses/users", routes.getUsers)
```

Comment:
Looks great so far! Some of these requests however are leaking sensitive information, e.g. ips of lecture halls. Maybe we can have a look at this together @MatthiasReumann ?

Reply:
Great catch. Forgot about that. 
Yes. 👍🏻